tests/unit/flow/test_flow_index.py

```diff
@@ -1,76 +0,0 @@
-import os
-import time
-
-import pytest
-
-from jina import Document
-from jina.flow import Flow
-from tests import random_docs, validate_callback
-
-cur_dir = os.path.dirname(os.path.abspath(__file__))
-
-
-def random_queries(num_docs, chunks_per_doc=5):
-    for j in range(num_docs):
-        d = Document()
-        d.id = j
-        for k in range(chunks_per_doc):
-            dd = Document()
-            dd.id = num_docs + j * chunks_per_doc + k
-            d.chunks.add(dd)
-        yield d
-
-
-@pytest.fixture
-def docpb_workspace(tmpdir):
-    os.environ['TEST_DOCSHARD_WORKSPACE'] = str(tmpdir)
-    yield
-    del os.environ['TEST_DOCSHARD_WORKSPACE']
-
-
-@pytest.mark.skipif(
-    'GITHUB_WORKFLOW' in os.environ, reason='skip the network test on github workflow'
-)
-@pytest.mark.parametrize('restful', [False, True])
-def test_shards_insufficient_data(mocker, restful, docpb_workspace):
-    """THIS IS SUPER IMPORTANT FOR TESTING SHARDS
-
-    IF THIS FAILED, DONT IGNORE IT, DEBUG IT
```

Comment:
But why this test was added in the first place?

Reply:
These are legacy testing codes. If the similar tests have been done somewhere else, feel free to deleteit